enclave-manager/server/server.go

```diff
@@ -89,12 +89,29 @@ func (c *WebServer) GetServiceLogs(
 	req *connect.Request[kurtosis_engine_rpc_api_bindings.GetServiceLogsArgs],
 	str *connect.ServerStream[kurtosis_engine_rpc_api_bindings.GetServiceLogsResponse],
 ) error {
+
 	result, err := (*c.engineServiceClient).GetServiceLogs(ctx, req)
 	if err != nil {
 		return err
 	}
-	res := result.Msg()
-	return str.Send(res)
+
+	logs := getServiceLogsFromEngine(result)
+	for {
+		select {
+		case <-ctx.Done():
+			err := result.Close()
+			if err != nil {
+				logrus.Errorf("Error ocurred: %+v", err)
```

Comment:
@Peeeekay minor thing but maybe we `stack.propagate` this error rather than logging here. That way the caller of this function can figure out what to do with it

Reply:
Saw your comment further down. Would you mind making the change in this PR?